packages/application-components/src/components/tabular-main-page/tabular-main-page.tsx

```diff
@@ -0,0 +1,63 @@
+import type { ReactElement, ReactNode } from 'react';
+import Spacings from '@commercetools-uikit/spacings';
+import { sharedMessages } from '@commercetools-frontend/i18n';
+import { css } from '@emotion/react';
+import { customProperties } from '@commercetools-uikit/design-system';
+import PageHeaderTitle from '../internals/page-header-title';
+import {
+  ControlsContainter,
+  TabularPageContainer,
+} from '../internals/tabular-page';
+import { ContentWrapper, PageWrapper } from '../internals/page.styles';
+
+type TTabularMainPageProps = {
+  /**
+   * The title of the page.
+   */
+  title: string;
+  /**
+   * The subtitle of the page.
+   */
+  subtitle?: string | ReactElement;
+  /**
+   * Replaces the title/subtitle row with a custom one (for special use cases)
+   */
+  customTitleRow?: ReactNode;
+  /**
+   * Any React node displayed as the tab panel.
+   */
+  children: ReactNode;
+  /**
+   * A composition of tab components.
+   */
+  tabControls: ReactNode;
+};
+
+const TabularMainPage = (props: TTabularMainPageProps) => (
+  <PageWrapper>
+    <TabularPageContainer color="surface">
```

Comment:
Maybe we need to allow the consumers to provide the color.
What do you think?

Reply:
Thanks. Let's ask @FilPob when he's back 🙂

Reply:
This `TabularPageContainer` is only used internally. We discussed that we will have 2 components to cover the 2 use cases: main page and details page. Those have predefined colors but the user does not have to do anything about it.

Reply:
Ok, I think I got something wrong because I understood that we wanted to have a design where the background colors could be inverted.
Default:
![Screenshot 2022-03-18 at 10 01 22](https://user-images.githubusercontent.com/97907068/158972915-63885053-0d43-471e-9047-78dc5eadad45.png)


Alternative:
![Screenshot 2022-03-18 at 10 02 17](https://user-images.githubusercontent.com/97907068/158972867-97eedcd4-a9de-430b-ad5e-4208904d46f1.png)

Apologize for my mistake.